packages/build/package.json

```diff
@@ -66,13 +66,13 @@
     "@bugsnag/js": "^7.0.0",
     "@netlify/cache-utils": "^5.1.3",
     "@netlify/config": "^20.3.6",
-    "@netlify/edge-bundler": "8.12.1",
+    "@netlify/edge-bundler": "8.12.2",
     "@netlify/framework-info": "^9.8.5",
     "@netlify/functions-utils": "^5.2.1",
     "@netlify/git-utils": "^5.1.1",
     "@netlify/plugins-list": "^6.66.0",
     "@netlify/run-utils": "^5.1.0",
-    "@netlify/zip-it-and-ship-it": "^8.9.0",
+    "@netlify/zip-it-and-ship-it": "8.9.0",
```

Comment:
Did you mean to lock the version of zip-it-and-ship-it as part of this?

Reply:
Yeah we do get PRs for updates anyway and using an older build version with a newer zisi version doesn't really make sense IMHO. Similar to what we do with edge-bundler.